boxoffice/mailclient.py

```diff
@@ -20,9 +20,13 @@ def send_receipt_mail(order_id, subject="Thank you for your order!"):
         order = Order.query.get(order_id)
         msg = Message(subject=subject, recipients=[order.buyer_email], bcc=[order.organization.contact_email])
         line_items = LineItem.query.filter(LineItem.order == order, LineItem.status == LINE_ITEM_STATUS.CONFIRMED).order_by("line_item_seq asc").all()
-        html = email_transform(render_template('order_confirmation_mail.html.jinja2', order=order, org=order.organization,
+        order_confirmed_amount = order.get_amounts(LINE_ITEM_STATUS.CONFIRMED).confirmed_amount
+        template = 'order_confirmation_mail.html.jinja2'
+        if(order_confirmed_amount == 0):
+            template = 'free_order_confirmation_mail.html.jinja2'
+        html = email_transform(render_template(template, order=order, org=order.organization,
             line_items=line_items,
-            order_confirmed_amount=order.get_amounts(LINE_ITEM_STATUS.CONFIRMED).confirmed_amount,
+            order_confirmed_amount=order_confirmed_amount,
```

Comment:
Use `order.paid_amount` if it's a first-time notification (since refund will be `0), use `order.net_amount` in other cases.

Reply:
Fixed